src/index.html

```diff
@@ -15,8 +15,31 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="dfsd">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li class="link is-active">APPLE</li>
```

Comment:
The text inside navigation should be wrapped in `<a>` tags to make them clickable links. Also, remember to use a proper `href` attribute to link to the corresponding sections of the page.